flaskdeid/deid.py

```diff
@@ -0,0 +1,32 @@
+import json
+import logging
+
+from flask import Blueprint, render_template, request, session, abort, jsonify, Response, current_app, g
+from flaskdeid import medlpInterface
+import medlp
+import hutchner
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+bp = Blueprint('deid', __name__, url_prefix='/deid')
+
+@bp.route("/id_phi", methods=['POST'])
+def identity_phi(**kwargs):
+
+    medlp_phi = medlp.annotate_phi(**kwargs)
+    hutchner_phi = hutchner.annotate_phi(**kwargs)
+    merged_results = _merge_phi(medlp_phi, hutchner_phi)
+    return merged_results
+
+def resynthesize_text(**kwargs):
```

Comment:
I think the resynthesis step is later in the pipeline, after the BRAT stuff?

Not sure if we'll use another endpoint on this same service or a completely different flask service, but either way, I think it'll involve sending the PHI annotations along with the text to the endpoint, rather than calling the `identify_phi` a second time (which not only is duplicated effort, but could end up with a different PHI annotation set than what was previously identified in the first PHI ID step).

Might be best to just leave the resynth stuff out for now, and we can figure out how it fits in later, once we complete the BRAT stuff (and also when Emily's back).

Reply:
That's fair. I've seperated it out as a second endpoint.